kzz7yh-f53797/cod-ve09v2_kzz7yh-f53797.xml

```diff
@@ -511,8 +514,8 @@
       <lb ed="#V" n="35"/>de costa secundum rationem causalem et primordialem. 
     </p>
                <p xml:id="kzz7yh-f53797-d1e964">
-                  <lb ed="#V" n="36"/>Ad primum in oppositum dicendum: quod potentam 
-      se<lb ed="#V" n="37" break="no"/>minaliter mundo inditam. Aug. ibi 
+                  <lb ed="#V" n="36"/>Ad primum in oppositum dicendum: quod potentiam 
+      se<lb ed="#V" n="37" break="no"/>minaliter mundo inditam. <ref>Aug.</ref> ibi 
```

Comment:
I would include "ibi" as part of the `<ref>` since we're sort of saying "Augustine in this place" says. And we'll have to look at context to figure what "place/ibi" is referring to

```suggestion
      se<lb ed="#V" n="37" break="no"/>minaliter mundo inditam. <ref>Aug. ibi</ref> 
```